lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -265,7 +261,7 @@ def _get_server_cert(server_cert_getter, self, charm):
         server_cert = server_cert_getter(self)
 
     if server_cert is None:
-        logger.warning(
+        logger.debug(
             f"{charm}.{server_cert_getter} returned None; continuing with INSECURE connection."
         )
```

Comment:
Is this logline needed?

Reply:
if we are given a cert getter, but the cert getter returns no cert, I think we should log something perhaps even at warning, because we might be transferring sensitive data in the clear, for all we know

Reply:
if we are not given a cert getter, then we assume the owner is fine with insecure connections

Reply:
Indeed we go into this flow only if cert getter is defined. We might want to stay at warning here then:
```suggestion
        logger.warning(
            f"{charm}.{server_cert_getter} returned None; sending traces over INSECURE connection."
        )
```